src/components/partials/IdentitySearch/CustomNodeModal.vue

```diff
@@ -102,9 +102,17 @@ export default class CustomNodeModal extends Vue {
         this.newCustomNodeAddress = this.customNode?.address ?? "";
     }
 
+    validateCustomNodeAddress() {
+        return this.newCustomNodeAddress.startsWith("wss://");
```

Comment:
Could you please also add a check for ws:// and allow that? The error message and placeholder can stay as is.